services/language_selector.rb

```diff
@@ -0,0 +1,15 @@
+class LanguageSelector
```

Comment:
Since all these classes are located in `services/` you can wrap all of them into `class YourName` namespace.
In this case, you can put inside shared constants, methods, etc. 
For example 
```ruby
def check_for_file_presence(path)
  unless File.exist?(path)
    puts I18n.t(:file_not_found)
    return
  end
end
```

Reply:
Added in another PR